core/lib/zksync_core/Cargo.toml

```diff
@@ -83,6 +83,8 @@ actix-web = "4.0.0-beta.8"
 
 tracing = "0.1.26"
 
```

Comment:
do we need this?

Reply:
yes there are still files within `zksync_core` that rely on it